src/jni/generated_jni_interface.jl

```diff
@@ -459,13 +459,13 @@ monitor_enter(obj::jobject, penv=jnienvptrs[]) =
 monitor_exit(obj::jobject, penv=jnienvptrs[]) =
   ccall(jninativeinterfaceref[].MonitorExit, jint, (Ptr{JNIEnv}, jobject,), penv, obj)
 
-get_java_v_m(vm::Array{JavaVM,1}, penv=jnienvptrs[]) =
+get_java_vm(vm::Array{JavaVM,1}, penv=jnienvptrs[]) =
   ccall(jninativeinterfaceref[].GetJavaVM, jint, (Ptr{JNIEnv}, Array{JavaVM,1},), penv, vm)
 
 get_string_region(str::jstring, start::Integer, len::Integer, buf::Array{jchar,1}, penv=jnienvptrs[]) =
   ccall(jninativeinterfaceref[].GetStringRegion, Nothing, (Ptr{JNIEnv}, jstring, jsize, jsize, Ptr{jchar},), penv, str, start, len, buf)
 
-get_string_u_t_f_region(str::jstring, start::Integer, len::Integer, buf::AbstractString, penv=jnienvptrs[]) =
+get_string_utfregion(str::jstring, start::Integer, len::Integer, buf::AbstractString, penv=jnienvptrs[]) =
```

Comment:
minor: should result in `get_string_utf_region`

Reply:
This is automatically generated and we consider multiple uppercase letters in a row as a single one. As such UTFRegion goes to utfregion. If we separate the last uppercase letter we will have the problem of ID becoming i_d again. A possible solution would be not to generate all the methods and just create the ones we actually need.